### PR TITLE
Add Keycloak Testcontainers community module for Golang

### PR DESCRIPTION
**Pull request by stillya**
Adds keycloak entry for Golang.
Based on this [issue](https://github.com/stillya/testcontainers-keycloak/issues/1)

### CODE REVIEW

File: modules/keycloak.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  - other
+docs:
+  - id: go
+    url: https://github.com/stillya/testcontainers-keycloak
+    isThirdParty: true
+    example: |
+      ```golang
+       ctx := context.Background()
+       container, err := keycloak.RunContainer(ctx,
+         testcontainers.WithWaitStrategy(wait.ForListeningPort("8080/tcp")),
```
Comment (eddumelendez) [inline modules/keycloak.md:13]: I think this should be an improvement in the module itself instead of define it every time.

**Review by eddumelendez — COMMENTED** [review]
Thanks for the PR, @stillya ! We already have this `keycloak` in our site but missing the one for `go`. I have added it here, LMK if [this](https://deploy-preview-128--testcontainers-site.netlify.app/modules/keycloak/) looks good. Otherwirse, please raise an issue in this repo.

**Comment by eddumelendez** [comment]
I'm closing this due to keycloak is already in our internal repository.

File: modules/keycloak.md (hunk 1 above)
Comment (stillya) [reply modules/keycloak.md:13]: Yeah, I'm going to add default wait strategy and others small improvements in the next release. Thanks.

**Pull request closed without merging**